bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestTruncateTable.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.hbase;
+
+import static com.google.cloud.bigtable.hbase.IntegrationTests.COLUMN_FAMILY;
+import static org.junit.Assert.*;
+
+import java.io.IOException;
+
+import org.apache.hadoop.hbase.TableName;
+import org.apache.hadoop.hbase.client.Admin;
+import org.apache.hadoop.hbase.client.Get;
+import org.apache.hadoop.hbase.client.Put;
+import org.apache.hadoop.hbase.client.Table;
+import org.junit.Test;
+
+public class TestTruncateTable extends AbstractTest {
+
+  @Test
+  public void testTruncate() throws IOException {
+    TableName tableName = IntegrationTests.newTestTableName();
+    IntegrationTests.createTable(tableName);
+    Admin admin = getConnection().getAdmin();
+    try (Table table = getConnection().getTable(tableName)) {
+      byte[] rowKey = dataHelper.randomData("testrow-");
+      byte[] qual = dataHelper.randomData("qual-");
+      byte[] value = dataHelper.randomData("value-");
+      Put put = new Put(rowKey);
+      put.addColumn(COLUMN_FAMILY, qual, 1L, value);
+      put.addColumn(COLUMN_FAMILY, qual, 2L, value);
+      table.put(put);
+      assertTrue(table.exists(new Get(rowKey)));
+      admin.disableTable(tableName);
```

Comment:
We have to add disableTable() before truncateTable() to make sure this test passes in hbase mode.